stable/kanister-operator/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
```

Comment:
This adds a dependency to an external chart. I'd like to discuss this with other maintainers first.

I noticed they copied this chart to their own repo, but the chart seems to be maintained here while you want to add a dependency to another chart from them. Seems kind of messy.

https://github.com/kanisterio/kanister/tree/master/helm/kanister-operator

Since you seem to contribute to kanister as well, could you clear up the picture?

Reply:
Hi @unguiculus, thanks for your questions. 
Here are only 2 maintainers me and @tdmanv. And we are both one of the main contributors to kanister. 

Why are we adding a new dependency?
Kanister requires profiles for some of the operations. Profile is a combinations of Profile Custome Resource, Sercret[s] and ConfigMap[s]. To make thigns ease we desided to create the profile chart. This will allow customers to create profiles while deploying kanister-operator, apps or independatly. 

Why this chart is stored in kansiter.charts repo?
Mainly because it's a helper chart and it doens't look like it belongs in k8s/charts
Also @mattfarina mentioned that community charts would support trusted repos, and kanister woudl liove to get into this list.

Why kanister-operator chart was copied into kasniter chart repo?
We had a cranch time and needed to release new verison urgently. Unfortunatly due amount of PRs,  merge into community charts repo takes a few days :( 
So we decided to host it our self and push an update to k8s/charts repo. 
I well be syncing these 2 charts, untill kasniter repo will be added as trusted repo into community repo. 

Hope this clears things up.

Reply:
Thanks for the explanations @depohmel - as we discussed on the call (not sure if you were there for that), the requirements in the stable repo should only point to charts in the stable repository (as per https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements).

Can I suggest, instead of contributing `profile` as a separate chart in stable, embedding the profile chart in the `charts/` directory of this chart. That way you are able to include this dependency without making it a top-level chart.

Reply:
@prydonius, no, I missed that part :(  
I'll add profile chart into /charts